Test/git-issues/git-issue-3804.dfy

```diff
@@ -0,0 +1,64 @@
+// RUN: %exits-with 4 %baredafny verify %args "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+predicate P(x: int)
+
+function FirstAssertNotProved(): (i: int)
+  ensures P(i)
+{
+  assert rule2: P(12) by {
```

Comment:
This assertion should generate a verification error (see my comment on L20 below).

Reply:
It does, so I just added a comment on the line of the test to be consistent.